### PR TITLE
FAQUse: What is difference anonset and anonscore

### PR DESCRIPTION
**Pull request by MarnixCroes**

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -892,6 +892,17 @@ It is different than anonymity set.
 For example, if the outputs are [1, 1, 1, 1, 0.5, 0.5, 0.5, 0.5, 0.4, 0.4, 0.2, 0.2, 0.2, 0.2, 0.2, 0.1, 0.1, 0.1, 0.1, 0.1] then, even when each of those 1s have anonscore 4, there are still lots of combinations of outputs that sum up to 1, so the anonset should be much higher but the anonscore is extremely conservative and takes into account many other things.
 :::
 
+:::details
+### What is the difference between anonset and anonscore?
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:896]:
```suggestion
### What is the difference between anonymity set and anonymity score?
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -892,6 +892,17 @@ It is different than anonymity set.
...
+
+The anonset and anonscore have minimal differences.
+The anonscore evolved into existence from the anonset, but with minimal changes, so they can be considered roughly the same thing.
+What differs is the kind of coinjoins they are applied to, where anonset was for Wasabi 1 and anonscore is for current Wasabi 2.
+In Wasabi 1 coinjoins there are unequal inputs, some equal outputs, and change coins. 
+In Wasabi 2 coinjoins there are many equal inputs, equal outputs, and (sometimes) some unique value outputs, but that does not mean that these unique value outputs are a change coin.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:902]:
```suggestion
In Wasabi 2 coinjoins there are many equal inputs, many equal outputs, and (sometimes) some unique value outputs, but that does not mean that these unique value outputs are change coins. 
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 3]
```diff
@@ -892,6 +892,17 @@ It is different than anonymity set.
...
 
+:::details
+### What is the difference between anonset and anonscore?
+
+The anonset and anonscore have minimal differences.
+The anonscore evolved into existence from the anonset, but with minimal changes, so they can be considered roughly the same thing.
+What differs is the kind of coinjoins they are applied to, where anonset was for Wasabi 1 and anonscore is for current Wasabi 2.
+In Wasabi 1 coinjoins there are unequal inputs, some equal outputs, and change coins. 
+In Wasabi 2 coinjoins there are many equal inputs, equal outputs, and (sometimes) some unique value outputs, but that does not mean that these unique value outputs are a change coin. 
+In fact most of the times it's not a change, but a private well mixed coin.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:903]:
```suggestion
In fact most of the times they are well mixed coins.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Review by yahiheb — COMMENTED** [review]
> The anonset and anonscore have minimal differences.
The anonscore evolved into existence from the anonset, but with minimal changes, so they can be considered roughly the same thing.

For me this kind of contradicts what we mention here:

https://github.com/zkSNACKs/WasabiDoc/blob/master/docs/FAQ/FAQ-UseWasabi.md#what-is-the-anonymity-score

> It is different than anonymity set.

File: docs/FAQ/FAQ-UseWasabi.md [hunk 4]
```diff
@@ -892,6 +892,17 @@ It is different than anonymity set.
 For example, if the outputs are [1, 1, 1, 1, 0.5, 0.5, 0.5, 0.5, 0.4, 0.4, 0.2, 0.2, 0.2, 0.2, 0.2, 0.1, 0.1, 0.1, 0.1, 0.1] then, even when each of those 1s have anonscore 4, there are still lots of combinations of outputs that sum up to 1, so the anonset should be much higher but the anonscore is extremely conservative and takes into account many other things.
 :::
 
+:::details
+### What is the difference between anonymity set and anonymity score?
+
+The anonset and anonscore have minimal differences.
+The anonscore evolved into existence from the anonset, but with minimal changes, so they can be considered roughly the same thing.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:899]:
```suggestion
The anonscore evolved into existence from the anonset, but with some differences.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Comment by yahiheb** [comment]
> remove `so they can be considered roughly the same thing.` ?

Done.

I am honestly still not satisfied with this one, because a couple of lines above we mention this when describing `What is the anonymity score?`

> It is different than anonymity set.

**Comment by yahiheb** [comment]
@kiminuo 

**Anonymity set:**

> The anonymity set is effectively the size of the group you are hiding in.
> If 3 people take part in a CoinJoin and there are 3 equal outputs then each of those output coins has an anonymity set of 3.

**Anonymity score:**

> The anonymity score is a way to estimate the level of entropy of a UTXO in an unequal-but-highly-composable output value coinjoin.
> It is different than anonymity set. 
> For example, if the outputs are [1, 1, 1, 1, 0.5, 0.5, 0.5, 0.5, 0.4, 0.4, 0.2, 0.2, 0.2, 0.2, 0.2, 0.1, 0.1, 0.1, 0.1, 0.1] then, even when each of those 1s have anonscore 4, there are still lots of combinations of outputs that sum up to 1, so the anonset should be much higher but the anonscore is extremely conservative and takes into account many other things.

Let's say you Kimi participated in the above coinjoin example with one input worth 1 btc and you received one output worth 1 btc. Your output's anonscore is 4 but for an outside observer they don't know whether you received one 1 btc output or two 0.5 outputs or one 0.5 + one 0.4 + one 0.1 btc outputs or ...
There are many combinations that could sum up to your 1 btc input.

One other thing to consider:

> Anon score is a conservative version of anon set, because in ww2 we cant assume each equal denomination output is owned by different entity.
> In ww2 you can be 7/8 of 0.1 BTC outputs. In ww1 you couldn't.
> Conservativeness comes from the wallet estimating your privacy based on the info it has about you, not based on what does an outsider see.
> Client knows it owns 7/8 outputs of same denomination, so it will give itself very small privacy bump if any. Maybe anon score 2.
> The client who had 1/8 outputs of that denomination will estimate anon set of 8 for his coin, but because it cant assume all of the UTXOs came from different entities, the client gives this coin maybe anon score 4.
> Same tx and different clients will calculate anon scores totally differently. They dont consider what kind of a clusterfuck does the outsider see.

**So basically anonymity score is a very conservative version of anonymity set because of:
1- The many combinations that are not taken into consideration.
2- In WW2 we cant assume each equal denomination output is owned by different entity.**

File: docs/FAQ/FAQ-UseWasabi.md [hunk 5]
```diff
@@ -892,6 +892,16 @@ It is different than anonymity set.
 For example, if the outputs are [1, 1, 1, 1, 0.5, 0.5, 0.5, 0.5, 0.4, 0.4, 0.2, 0.2, 0.2, 0.2, 0.2, 0.1, 0.1, 0.1, 0.1, 0.1] then, even when each of those 1s have anonscore 4, there are still lots of combinations of outputs that sum up to 1, so the anonset should be much higher but the anonscore is extremely conservative and takes into account many other things.
 :::
 
+:::details
+### What is the difference between anonymity set and anonymity score?
+
+The anonscore evolved into existence from the anonset, but with some differences.
+What differs is the kind of coinjoins they are applied to, where anonset was for Wasabi 1 and anonscore is for current Wasabi 2.
+In Wasabi 1 coinjoins there are unequal inputs, some equal outputs, and change coins. 
+In Wasabi 2 coinjoins there are many equal inputs, many equal outputs, and (sometimes) some unique value outputs, but that does not mean that these unique value outputs are change coins.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:901]:
```suggestion
In Wasabi 2 coinjoins there are many equal inputs, many equal outputs, and (sometimes) some unique value outputs, that are not necessarily change coins.
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 6]
```diff
@@ -892,6 +892,16 @@ It is different than anonymity set.
...
 :::
 
+:::details
+### What is the difference between anonymity set and anonymity score?
+
+The anonscore evolved into existence from the anonset, but with some differences.
+What differs is the kind of coinjoins they are applied to, where anonset was for Wasabi 1 and anonscore is for current Wasabi 2.
+In Wasabi 1 coinjoins there are unequal inputs, some equal outputs, and change coins. 
+In Wasabi 2 coinjoins there are many equal inputs, many equal outputs, and (sometimes) some unique value outputs, but that does not mean that these unique value outputs are change coins. 
+In fact most of the times they are well mixed coins.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:902]:
```suggestion
In fact most of the times they are well mixed coins.

Basically the anonymity score is an extremely conservative anonymity set calculation.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Review by yahiheb — DISMISSED** [review]

**Review by yahiheb — APPROVED** [review]
